pkg/gopro/gopro.go

```diff
@@ -1046,3 +732,53 @@ func getFileTime(osPathname string, utcFix bool) time.Time {
 	}
 	return d
 }
+
+func getDayFolder(cameraName string, output string, sortoptions SortOptions, mediaDate string) string {
+	dayFolder := filepath.Join(output, mediaDate)
+	if _, err := os.Stat(dayFolder); os.IsNotExist(err) {
+		_ = os.Mkdir(dayFolder, 0755)
+	}
+
+	if sortoptions.ByCamera {
+		if _, err := os.Stat(filepath.Join(dayFolder, cameraName)); os.IsNotExist(err) {
+			_ = os.Mkdir(filepath.Join(dayFolder, cameraName), 0755)
+		}
+		dayFolder = filepath.Join(dayFolder, cameraName)
+	}
+        return dayFolder
+}
+
+func getMediaDate(d time.Time, sortoptions SortOptions) string {
+	mediaDate := d.Format("02-01-2006")
+	if strings.Contains(sortoptions.DateFormat, "yyyy") && strings.Contains(sortoptions.DateFormat, "mm") && strings.Contains(sortoptions.DateFormat, "dd") {
+		mediaDate = d.Format(replacer.Replace(sortoptions.DateFormat))
+	}
+        return mediaDate
+}
+
+func parse(folder string, name string, de *godirwalk.Dirent, osPathname string, sortoptions SortOptions, result utils.Result) utils.Result {
```

Comment:
`de *godirwalk.Dirent` is unused in the function.

Reply:
Thanks, removed